LA4Ice.c

```diff
@@ -283,7 +283,7 @@ int main(int argc, char *argv[])
   }
 
   //  Read the file and display selected records
-  
+  if (tspace > 0)
```

Comment:
Thanks.  Do you think I should merge this pull request? Or wait for you to fix them all together.